src/components/Pagination/Pagination.tsx

```diff
@@ -121,6 +132,7 @@ export const Pagination = ({
           pageLabel={pageLabel}
           pages={pages}
           currentPage={currentPage}
+          tracking={tracking}
```

Comment:
We want to track the page number that is clicked. We can have this value only from the arguments of onChange function. We need to take it out from there and put it to label field. Our useClickTracker doesn't support this. So an alternative is to use the tracking of buttons, like here.